JapaneseFilter/sections/specific.txt

```diff
@@ -18,6 +18,10 @@ gigafile.nu###shinkarionz
 gigafile.nu###geniee_overlay_outer
 altema.jp,gorillawiki.jp##.new_koukokuwaku_wrapper
 mainichi.jp###index-remarkable
+touhouyaruosure.com##.footer ~ div.main a[href^="https://"][rel]
```

Comment:
Please exchange this rule to `touhouyaruosure.com##.footer ~ div.main a[href^="https://"]:not([href*="twitter.com/"])`